core/src/test/java/tech/tablesaw/io/csv/CsvReaderTest.java

```diff
@@ -716,6 +774,7 @@ public void testReadCsvWithRowSampling() throws IOException {
   }
 
   @Test
+  @SuppressWarnings("unchecked")
```

Comment:
Can you remove this line? I just fixed it so that it's no longer necessary

Reply:
Fixed now.